libopae/src/enum.c

```diff
@@ -338,10 +338,10 @@ bool del_dev(struct dev_list *pdev, struct dev_list *parent)
 	return true;
 }
 
-static const fpga_guid FPGA_FME_GUID = {
-	0xbf, 0xaf, 0x2a, 0xe9, 0x4a, 0x52, 0x46, 0xe3,
-	0x82, 0xfe, 0x38, 0xf0, 0xf9, 0xe1, 0x77, 0x64
-};
+//static const fpga_guid FPGA_FME_GUID = {
```

Comment:
What is the warning that commenting out this code fixes?  Why not just delete this code?  If there is a reason to keep it then please add a comment explaining why.

Reply:
@southerngs see line 381 in the same file.